src/app/components/question-list/question-list.component.html

```diff
@@ -48,24 +69,12 @@ <h2>Lista wszystkich pytań. <h3 *ngIf="isButtonsEnabled() === false">Możesz us
         </div>
       </mat-expansion-panel>
     </mat-accordion>
-    <div *ngIf="isButtonsEnabled()" class="add-question">
+    <div *ngIf="isQuizAdding() === false && isButtonsEnabled() == false" class="add-question">
```

Comment:
w tych metodach typu `isQuizAdding()` zwracasz od razu pole komponentu a tutaj porównujesz. to juz lepiej bezpośrednio porównać tutaj pole albo wszystko robic w metodzie.

Reply:
i to sie tyczy kązdego takiego ngIfa